pkg/ingester/client/ingester.proto

```diff
@@ -54,6 +54,7 @@ message LabelValues {
 message LabelValuesCardinalityRequest {
   repeated string label_names = 1;
   repeated LabelMatcher matchers = 2;
+  bool active_series_only = 3;
```

Comment:
Will this change based on the survey you opened? I know the survey is for the HTTP API, but now we have the chance to make gRPC and HTTP align.

Reply:
Same question applies for `UserStatsRequest`?

Reply:
Sure, I think it's a good idea to make them match, I'm planning to make a final decision tomorrow, then I'll take a last look at the grpc api to make sure it makes sense.